generator/src/main/java/com/scottlogic/deg/generator/commandline/GenerateCommandLine.java

```diff
@@ -209,6 +216,11 @@ public boolean visualiseReductions() {
         return outputFormat;
     }
 
+    @Override
+    public GenerationConfig.OutputDestination getOutputDestination() {
+        return outputDestination;
```

Comment:
This can be derived from whether the output filename is empty or not.

```suggestion
        return outputPath == null ? OutputDestination.STREAM : OutputDestination.FILE;
```